Index: openwrt-src/target/linux/ramips/base-files/lib/ramips.sh
===================================================================
--- openwrt-src.orig/target/linux/ramips/base-files/lib/ramips.sh	2016-09-12 17:03:56.549906014 +0800
+++ openwrt-src/target/linux/ramips/base-files/lib/ramips.sh	2016-09-12 23:47:10.593255293 +0800
@@ -387,6 +387,12 @@
 		;;
 	*"HG255D")
 		name="hg255d"
+                ;;
+	*"Huawei HG255D")
+		name="hg255d"
+                ;;
+	*"HuaWei HG255D")
+		name="hg255d"
 		;;
 	*"V22SG")
 		name="v22sg"
Index: openwrt-src/target/linux/ramips/base-files/etc/diag.sh
===================================================================
--- openwrt-src.orig/target/linux/ramips/base-files/etc/diag.sh	2016-09-12 17:03:56.549906014 +0800
+++ openwrt-src/target/linux/ramips/base-files/etc/diag.sh	2016-09-12 17:03:56.549906014 +0800
@@ -81,6 +81,9 @@
 	hlk-rm04)
 		status_led="hlk-rm04:red:power"
 		;;
+	hg255d)
+		status_led="hg255d:power"
+		;;
 	ht-tm02)
 		status_led="ht-tm02:blue:wlan"
 		;;
