--- a/openwrt-src/target/linux/brcm47xx/config-3.18
+++ b/openwrt-src/target/linux/brcm47xx/config-3.18
@@ -35,7 +35,7 @@ CONFIG_BCMA_SFLASH=y
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 CONFIG_CEVT_R4K=y
 CONFIG_CLONE_BACKWARDS=y
-CONFIG_CMDLINE="noinitrd console=ttyS0,115200"
+CONFIG_CMDLINE="noinitrd console=/dev/null"
 CONFIG_CMDLINE_BOOL=y
 # CONFIG_CMDLINE_OVERRIDE is not set
 # CONFIG_CPU_BMIPS is not set
